cli/integration_tests/single_package/setup.sh

```diff
@@ -4,3 +4,10 @@ SCRIPT_DIR=$(dirname ${BASH_SOURCE[0]})
 TARGET_DIR=$1
 cp -a ${SCRIPT_DIR}/my-pkg/. ${TARGET_DIR}/
 ${SCRIPT_DIR}/../setup_git.sh ${TARGET_DIR}
+
+if [ "$2" != "" ]; then
+  pm="$2"
+  jq --arg pm "$pm" '.packageManager = $pm' "$TARGET_DIR/package.json"
+  # cat "$TARGET_DIR/package.json" | jq ".packageManager=\"$2\"" | sponge "$TARGET_DIR/package.json"
```

Comment:
Should the `cat` command be uncommented? Reading through `jq` docs, I don't think that command will update the `package.json`.

Reply:
`cat` wasn't working either becuase `sponge` apparently isn't installed in CI (I thought it was). So I was experimenting with using `jq` directly, and haven't been successful yet.

Reply:
finagled it a slightly different way